cn-india/middlewares/isAuth.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const User = require("../models/user");
+var jwt = require("jsonwebtoken");
```

Comment:
lease use const instead of var and stack all npm packages at top and all local models/files after them